pyproject.toml

```diff
@@ -37,6 +37,7 @@ dependencies = [
     "rich>=13.7.0,<14",
     "solc-select>=1.0.4,<2",
     "filelock>=3.15.1,<4",
+    "pytest-json-report>=1.5.0,<2",
```

Comment:
Just wondering where this is used?

Reply:
It's not used for this specific PR but I used to get the JSON output from the fill and make it easier for me to detect the failing duplicate containers (it enables the `--json-report` `--json-report-file` flags).

Reply:
Removed the commit :+1: